src/components/Connections/Connections.tsx

```
@@ -523,7 +528,37 @@ const SyncStatus = memo(
+  const handleCancelSync = useCallback(() => {
+    onCancelSync(connection.connectionId);
+  }, [connection.connectionId, onCancelSync]);
+
+  return (
+    <div>
+      <span>{connection.name}</span>
+      <button onClick={handleSync}>Sync</button>
+      {connection.isSyncing && <button onClick={handleCancelSync}>Cancel</button>}
+    </div>
+  );
+});
```

ESLint

[error] Component definition is missing display name

(react/display-name)